src-vue/src/views/DeveloperView.vue

```diff
@@ -29,6 +35,26 @@ export default defineComponent({
                 type: 'error',
                 position: 'bottom-right'
             });
+        },
+        async toggleReleaseCandidate() {
+            // Flip between RELEASE and RELEASE_CANDIDATE
+            if (this.$store.state.release_canal === ReleaseCanal.RELEASE) {
```

Comment:
This can be done with a [ternary operator](https://www.c-sharpcorner.com/UploadFile/5089e0/ternary-conditional-operator-in-typescript/) to be more readable.

```typescript
this.$store.state.release_canal = this.$store.state.release_canal === ReleaseCanal.RELEASE
    ? ReleaseCanal.RELEASE_CANDIDATE
    : ReleaseCanal.RELEASE;
```

Reply:
For code to be even more readable, we could put `release_canal` in the computed properties of the DeveloperView component and [add associated getter/setter](https://codingexplained.com/coding/front-end/vue-js/adding-getters-setters-computed-properties):

```typescript
computed: {
	release_canal: {
		get: function(): string {
      			return this.$store.state.release_canal;
		},
		set: function(newValue: string) {
			this.$store.state.release_canal = NorthstarReleaseType[newValue];
		}
	}
}
```

Which would enable writing:

```typescript
this.release_canal = this.release_canal === ReleaseCanal.RELEASE
    ? ReleaseCanal.RELEASE_CANDIDATE
    : ReleaseCanal.RELEASE;
```

This is up to you ;)

Reply:
I went with just the ternary operator in 05c945aaa13bbb20859ae4eb4eff98a0a1e91211 for the sole reason that simply copy-pasting the above snippet did not work as `NorthstarReleaseType` is not recognised :P